jobs/buildhub/inventory_to_records.py

```diff
@@ -403,6 +403,23 @@ class JSONFileNotFound(Exception):
         if result:
             yield result
 
+    def deduplicate_entries_in_folder(entries):
+        """
+        If there are two identical releases in the same folder, we keep the
+        ones that contain "setup" or "installer".
+        """
+        deduplicated = {}
+        for entry in entries:
+            identifier = entry['Key'].lower() \
+                                     .replace('+setup+', '-') \
+                                     .replace('.installer.exe', '') \
+                                     .replace('.exe', '')
+            dupped = deduplicated.get(identifier, {"Key": ""})
+            is_installer = 'installer' in dupped['Key'] or '+setup+' in dupped['Key']
```

Comment:
This will be true for `....installer-stub.exe` and `installer.tar.gz` too. 
Not sure that that means. Just makes me nervous.

Reply:
installer-stub will be filtered by is_build_url() I believe

Reply:
You are right, I will adjust this